rbac/internal/views.py

```diff
@@ -77,9 +84,40 @@
 )
 from api.utils import RESOURCE_MODEL_MAPPING, get_resources
 
+# environment variables
+HOST = settings.REDHAT_STAGE_SSO
+client_id = os.getenv("CLIENT_ID")
+client_secret = os.getenv("CLIENT_SECRET")
+scopes = settings.SCOPE
+url = settings.OPENID_URL
+grant_type = settings.TOKEN_GRANT_TYPE
+relations_api_server = settings.RELATION_API_SERVER
+
 logger = logging.getLogger(__name__)
 TENANTS = TenantCache()
 PROXY = PrincipalProxy()
+JWT = JWTCache()
+
+
+conn = None
+# Create http client
+if HOST is not None:
```

Comment:
can you put it in the little class and memoize it ?

look on ITService 

and I call constructor  with
```
http.client.HTTPSConnection(HOST)
```
only when it is needed  ? (when API is called )